src/main/Form/Form.java

```diff
@@ -417,6 +465,46 @@ public Question(
       this.conditionalType = conditionalType;
     }
 
+    public ObjectId getId() {
+      return id;
+    }
+
+    public FieldType getType() {
+      return type;
+    }
+
+    public String getQuestionText() {
+      return questionText;
+    }
+
+    public List<String> getOptions() {
+      return options;
+    }
+
+    public String getDefaultValue() {
+      return defaultValue;
+    }
+
+    public boolean isRequired() {
+      return required;
+    }
+
+    public int getNumLines() {
+      return numLines;
+    }
+
+    public boolean isMatched() {
+      return matched;
+    }
+
+    public ObjectId getConditionalOnField() {
```

Comment:
just curious why are you using ObjectId here as a return type rather than a Id of the Section? Or is it? I guess its a bit unclear. Maybe a comment would help